lib/src/controllers/chat/ds_audio_message_bubble.controller.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:get/get.dart';
+import 'package:just_audio/just_audio.dart';
+import 'package:rxdart/rxdart.dart' as rx_dart;
+
+class DSAudioMessageBubbleController extends GetxController {
+  final audioSpeed = RxDouble(1.0);
+  final audioSpeedLabel = RxString('x1');
```

Comment:
Since the label will always follow the same pattern "x<audioSpeed>", you could actually set this to use the audioSpeed value dinamically (i.e: "x$audioSpeed"). That way, the audioSpeedLabel doesn't need to be changed if, in future, we decide to create a new speed value, like 4x, for instance.

Reply:
Very good, I implemented as your suggestion